### DISCUSSION TITLE
Livewire 3 doesn't get along with pagination?

### DISCUSSION THREAD

**Discussion by Marre-86** [livewire/livewire#6070] [1. Help] [answered]
I have upgraded my simple pet project to Livewire 3 and get a weird error I cannot google a solution on.
In my Livewire component I have this:
```
        $activities = Activity::where('created_by_id', Auth::user()->id)
            ->bla-bla-bla
            ->paginate(15);
```
And it throws this error:
```
Livewire \ Exceptions \ PropertyNotFoundException 
Property [$paginators] not found on component: [search-activities]
```
In case I replace `paginate(15) `with `get()` it works like charm
Before I upgraded  to ver.3 it worked like charm.

Help please.

**Comment by romansh** [comment, marked as answer]
use Livewire\WithPagination;

**Reply by Marre-86** [reply]
It didn't help.
The same error

**Reply by Marre-86** [reply]
Actually it did help.
Just had to specify     `use WithPagination;` within the class itself too....
Don't know why it works even without it in V2.

**Comment by nuernbergerA** [comment]
You have to republish the paginator view files

**Reply by Marre-86** [reply]
Found the paragraph about it in official upgrading guide. But didn't quite get what exactly I am supposed to do.

**Reply by PhiloNL** [reply]
You can re-publish the views using the following command:
```shell
php artisan livewire:publish --pagination
```
